sdk/java-sdk/src/main/scala/kalix/javasdk/impl/action/ActionEffectImpl.scala

```diff
@@ -68,6 +65,13 @@ object ActionEffectImpl {
       copy(internalSideEffects = sideEffects)
   }
 
+  final case class EmptyEffect[T](internalSideEffects: Seq[SideEffect]) extends PrimaryEffect[T] {
```

Comment:
I vote for calling it Ignore all the way to protocol, so `IgnoreEffect`

Reply:
left over from the previous implementation. Yes, that is better

Reply:
done